Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -3808,9 +3829,11 @@ internal void RestoreStateWithoutDevices(SerializedState state)
                 Object.DestroyImmediate(m_Settings);
             m_Settings = state.settings;
 
-            if (m_Actions != null)
```

Comment:
This didn't make any sense to keep since when triggered it would attempt to destroy an action associated with an asset.